dlib/dnn/layers_abstract.h

```diff
@@ -573,6 +573,13 @@ namespace dlib
                 - #get_bias_weight_decay_multiplier() == val
         !*/
 
+        void disable_bias(
+        );
+        /*!
+            ensures
+                - disables bias for this layer
```

Comment:
Add a `bool bias_is_disabled()` method too so that someone can tell if a layer has had its bias disabled or not.  Then make the docs for the `disable_bias()` methods note that they set `bias_is_disabled()` to false.